### DISCUSSION TITLE
Failed to load resource: the server responded with a status of 500 (Internal Server Error)

### DISCUSSION THREAD

**Discussion by tiffanyiskwzj** [Netflix/conductor#2406] [General]
Use the pre-configured Docker image， I started three images, images tag namely ui、server、es，next  to view the UI, navigate to localhost:5000,Some menu url are inaccessible ,like menu Workflows,the url is ":5000/api/wfe/metadata/workflow"，I can't find the relevant server code whose url starts with “api/wfe ” in the code,I only found it in the ui code like 'server.js'. I don't know what I have done less.

thank you.

**Comment by apanicker-nflx** [comment]
Thanks for reporting this, we will investigate this and get back to you.
